src/util/styles.ts

```diff
@@ -1,11 +1,13 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-import { IFontWeight } from '@uifabric/styling/lib';
-import { ICSSPixelUnitRule } from '@uifabric/merge-styles/lib/IRawStyleBase';
+import { IRawStyle } from '@uifabric/styling';
 
-const borderRadiusSoft: ICSSPixelUnitRule = '2px';
-const fontWeightBold: IFontWeight = '600';
+export const borderRadiusSoft: IRawStyle = { borderRadius: '2px' };
+export const borderRadiusRound: IRawStyle = { borderRadius: '50%' };
+export const fontWeightBold: IRawStyle = { fontWeight: '600' };
+export const fontWeightNormal: IRawStyle = { fontWeight: '600' };
```

Comment:
Why export the object instead of just the value? It forces you to spread all over the place.

Reply:
It was an artifact of a previous iteration, where it got me past a typing challenge. I've update it to export values now.

Reply:
looks like error text color changed, was this on purpose?

Reply:
It was, but I was confused, undoing it now.